charts/temporal/values.yaml

```diff
@@ -242,6 +242,33 @@ server:
     containerSecurityContext: {}
     topologySpreadConstraints: []
     podDisruptionBudget: {}
+  internal-frontend:
```

Comment:
also, to be backwards compatible, should we disable internal frontend by default?

Reply:
@adamko147 agreed, that was meant to be disabled by default, I'll make this update